spec/controllers/videos_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+describe VideosController do
+  describe 'GET show' do
+    let(:video) { Fabricate(:video) }
+
+    context 'authenticated user' do
+      before do
+        session[:user_id] = Fabricate(:user).id
+      end
+      it 'sets @video' do
+        get :show, id: video.id
+        expect(assigns(:video)).to eq(video)
+      end
+      
+      it 'sets @reviews' do
+        review1 = Fabricate(:review, video: video)
+        review2 = Fabricate(:review, video: video)
+        get :show, id: video.id
+        assigns(:reviews).should =~ [review1, review2]
```

Comment:
you are using `should` syntax here and everywhere else `expect` syntax. Stick to 1 only to look coherent, and preferably the latest `expect` would be the way to go nowadays.